spec/new_inject_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'array'
+
+describe Array do
+
+	describe "#new_inject" do
+		it "returns the only element of array" do
+			
+			expect( [1].new_inject{ |mem, el| el } ).to eq 1
+		end
+
+		it "sum the numbers" do
+			expect( [ 1, 2 ].new_inject{ |ob, el| el + ob } ).to eq 3
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing to the left of {.<br>Space inside square brackets detected.<br>Space inside parentheses detected.<br>Tab detected.